src/Facebook/InstantArticles/Validators/Type.php

```diff
@@ -396,9 +398,10 @@ public static function isTextEmpty($text)
             return true;
         }
         // Stripes empty spaces, &nbsp;, <br/>, new lines
-        $text = strip_tags($text);
-        $text = preg_replace("/[\r\n\s]+/", "", $text);
```

Comment:
You've removed this \n and \r. Is this still working for tabs, <enters> etc?
If so, thats fine :-)